src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -31,18 +31,23 @@ import { CancellationOptions } from './CancellationOptions'
 const DateFormsWrapper = styled.div`
   display: flex;
   flex-direction: row;
+  flex-wrap: wrap;
 `
+type SplitData = 'left' | 'right' | false
```

Comment:
Isn't it clearer? Or something similar

```suggestion
type SplitData = 'left' | 'right' | 'full'
```